### ISSUE TITLE
Cython の調査

### ISSUE THREAD

**Issue by mrkn** [sciruby-jp/survey#9]
Cython の特徴を調査してください

**Comment by v0dro** [comment]
I don't think we an equivalent of Cython in Ruby. I'm thinking of taking that up as my next project though...

**Comment by mrkn** [comment]
@v0dro ko1 will investigate how to realize cython-like system in Ruby.

**Comment by v0dro** [comment]
Do keep me informed. I am deeply interested in such a project.

**Comment by mrkn** [comment]
@v0dro I recommend you to talk with @ko1 about that topic.
He is one of the best person to investigate and realize Cython-like system in Ruby.
So you should cooperate with him.
